lib/webhdfs.js

```diff
@@ -520,7 +520,7 @@ WebHDFS.prototype.readFile = function readFile (path, opts, callback) {
   var hdfsWatchdog = setTimeout(timeoutOperation, timeoutDuration);
 
   remoteFileStream.once('error', function onError (err) {
-    error = err;
+    if(err && err.port !== 50075) error = err;
```

Comment:
We should also check if the error code is `ECONNREFUSED`. You don't want to ignore legit errors thrown on 50075 port by `createReadStream`. For instance from here https://github.com/AlphonsoCode/webhdfs/blob/51b8d09fff41ad48ec607033181a2eebe7e099ad/lib/webhdfs.js#L728-L731

Reply:
done.